src/pages/form/basic-form/locales/zh-CN.ts

```diff
@@ -0,0 +1,59 @@
+export default {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of default exports is forbidden](https://app.codacy.com/app/afc163/ant-design-pro/pullRequest?prid=4583897)